### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by olenitut**
- [DEMO LINK](https://olenitut.github.io/layout_calendar/)
- [TEST REPORT LINK](https://olenitut.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,7 +6,43 @@
...
+        <li class="calendar__day calendar__day--15"></li>
+        <li class="calendar__day calendar__day--16"></li>
+        <li class="calendar__day calendar__day--17"></li>
+        <li class="calendar__day calendar__day--18"></li>
+        <li class="calendar__day calendar__day--19"></li>
+        <li class="calendar__day calendar__day--20"></li>
+        <li class="calendar__day calendar__day--21"></li>
+        <li class="calendar__day calendar__day--22"></li>
+        <li class="calendar__day calendar__day--23"></li>
+        <li class="calendar__day calendar__day--24"></li>
```
Comment (SanyaBratashchuk) [inline src/index.html]: Remove calendar__day--{{number}} class and use nth-child

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  min-width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__month {
+    min-width: calcWidth($column-width, $gap, $num-columns, $calendar-padding);
+    max-width: calcWidth($column-width, $gap, $num-columns, $calendar-padding);
+    padding: #{$calendar-padding};
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/calendar.scss:11]:
```suggestion
    padding: $calendar-padding;
```

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translate(0, -20px);
+    }
+
+    @for $day from 1 through 31 {
+      &:nth-child(#{$day}):before {
+        content: "#{$day}";
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/calendar.scss:45]: only here `#{}` make sense

File: src/index.html [hunk 4]
```diff
@@ -6,7 +6,43 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <main class="calendar">
+      <ul class="calendar__month calendar__month--month-length_28">
+        <li 
+        class="calendar__day calendar__day--1 calendar__day--start-day_sun"
```
Comment (ivanchuk-yurii) [inline src/index.html:13]: `calendar__day--1` is redundant

**Review by ivanchuk-yurii — APPROVED** [review]
comments aren't critical

**Pull request closed without merging**